app/experimenter/jetstream/tests/test_tasks.py

```diff
@@ -201,6 +202,10 @@ def open_file(filename):
         experiment = NimbusExperiment.objects.get(id=experiment.id)
         self.assertIsNone(experiment.results_data)
 
+        mock_get_metadata.return_value = {
+            "outcomes": {"default-browser": {"metrics": ["test"], "default_metrics": []}}
+        }
```

Comment:
Does this recreate the failure we see in prod?  Testing this locally, reverting the changes to `client.py` still lets the tests pass?

Ideally we'd modify/extend the tests to force the case we see in sentry:

```py
IndexError

list index out of range

    experimenter/jetstream/models.py in append_conversion_count at line 261

                conversion_count = 0.0
                if None not in (population_count, conversion_percent):
                    conversion_count = population_count * conversion_percent
                absolute_primary_metric_vals.first.count = conversion_count
                absolute_primary_metric_vals.all[0].count = conversion_count  <<<<<<<<<
```

Reply:
This fixes the existing test cases, but you're right -- it doesn't cover the failure fixed by this PR. I added another commit updating the tests and test data to cover this.